runtime/tests/interpreter/builtinfunctions_test.go

```diff
@@ -147,6 +149,53 @@ func TestInterpretToBytes(t *testing.T) {
 	})
 }
 
+func TestInterpretAddressFromBytes(t *testing.T) {
+
+	t.Parallel()
+
+	runAddressTest := func(t *testing.T, expected []byte, innerCode string) {
+		t.Run(innerCode, func(t *testing.T) {
+			t.Parallel()
+
+			code := fmt.Sprintf("fun test(): Address { \n return Address.fromBytes(%s) \n }", innerCode)
+
+			inter := parseCheckAndInterpret(t, code)
+			res, err := inter.Invoke("test")
+
+			require.NoError(t, err)
+
+			addressVal, ok := res.(interpreter.AddressValue)
+			require.True(t, ok)
+
+			require.Equal(t, expected, addressVal.ToAddress().Bytes())
+		})
+	}
+
+	runAddressTest(t, []byte{1}, "[1]")
+	runAddressTest(t, []byte{12, 34, 56}, "[12, 34, 56]")
+	runAddressTest(t, []byte{67, 97, 100, 101, 110, 99, 101, 33}, "[67, 97, 100, 101, 110, 99, 101, 33]")
+}
+
+func TestInterpretAddressFromBytesInvalid(t *testing.T) {
+
+	t.Parallel()
+
+	runAddressTest := func(t *testing.T, innerCode string) {
+		t.Run(innerCode, func(t *testing.T) {
+			t.Parallel()
+
+			code := fmt.Sprintf("fun test(): Address { \n return Address.fromBytes(%s) \n }", innerCode)
```

Comment:
Here and above: Using Go raw string literals (`) makes the code a bit more readable:

```suggestion
			code := fmt.Sprintf(`
                  fun test(): Address {
                      return Address.fromBytes(%s)
                  }
            	`,
				innerCode,
			)
```